Data/HashMap/Internal.hs

```diff
@@ -163,9 +163,9 @@ import Text.Read                  hiding (step)
 import qualified Data.Data                   as Data
 import qualified Data.Foldable               as Foldable
 import qualified Data.Functor.Classes        as FC
-import qualified Data.HashMap.Internal.Array as A
 import qualified Data.Hashable               as H
 import qualified Data.Hashable.Lifted        as H
+import qualified Data.HashMap.Internal.Array as A
 import qualified Data.List                   as List
```

Comment:
Why are you rearranging the imports?

Reply:
It looks like `stylish-haskell` does this?

Reply:
Please anti-do it. It's distracting.

Reply:
I can anti-do it. Is there some configuration option for `stylish-haskell` though, I can't seem to find it.